app/src/constants/addresses.ts

```diff
@@ -0,0 +1,14 @@
+export const DAO_TREASURY_ADDRESS = 'BRSda3A6o3czoPhsAEHjA5dZVrYL4uJ3JWXxwFSY9pJM';
+export const STAKING_OPTIONS_ADDRESSES = [
```

Comment:
Need to pull this list of accounts dynamically. They will change/expire over the course of time. Check out how we pull data into the SO (stakingOptions.tsx) tab as reference